data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/OutputFormat.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.processor.aggregate.actions;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum OutputFormat {
+
+    OTEL_METRICS("otel_metrics"),
+    DEFAULT("default");
```

Comment:
I'm not sure this should be named `default`. We might want `otel_metrics` to be default. I'm unsure of the best name here, but maybe `event`, `basic`, or `raw` would be better suited.

Reply:
I will make it "raw"